starterOnly/modal.js

```diff
@@ -27,3 +27,79 @@ function closeWindow() {
   }
 }
 
+// Form Variables
+const firstName = document.getElementById("first"),
+  lastName = document.getElementById("last"),
+  email = document.getElementById("email"),
+  birthdate = document.getElementById("birthdate"),
+  quantity = document.getElementById("quantity"),
+  country = document.getElementsByClassName("location"),
+  agreeToTerms = document.getElementById("checkbox1"),
+  form = document.getElementById("form"),
+  submit = document.getElementById("submit"),
+  errorMsg = document.getElementsByClassName("error");
+
+  // ---------------- Form Validation ----------------
+
+// ---- First Name Validation
+const checkFirstName = (id, serial, message) => {
+  errorMsg[serial].innerHTML =
+    id.value === "" || id.value.length < 2 ? message : "";
+  verification.FirstName =
+    id.value === "" || id.value.length < 2 ? false : true;
+};
+
+// ---- Last Name Validation
+const checkLastName = (id, serial, message) => {
+  errorMsg[serial].innerHTML =
+    id.value === "" || id.value.length < 2 ? message : "";
+  verification.LastName = id.value === "" || id.value.length < 2 ? false : true;
```

Comment:
revisit this condition?

Reply:
!(verification.LastName = id.value === "" || id.value.length < 2)

Reply:
Done